studio/components/to-be-cleaned/Storage/StorageUsage/StorageUsage.tsx

```diff
@@ -1,11 +1,14 @@
+import { FC } from 'react'
 import SVG from 'react-inlinesvg'
 
-const StorageUsage = () => {
+interface Props {}
+
+const StorageUsage: FC<Props> = () => {
   return (
     <div className="w-full h-full flex flex-col items-center justify-center">
       <SVG
         src={'/img/storage-usage.svg'}
-        alt={'storage-usage'}
+        title={'storage-usage'}
```

Comment:
Changed to `title` as per http://web-accessibility.carnegiemuseums.org/code/svg/